R/xml_find.R

```diff
@@ -93,6 +108,6 @@ xml_find_one.xml_nodeset <- function(x, xpath, ns = character()) {
   if (length(x) == 0)
     return(xml_nodeset())
 
-  nodes <- lapply(x, function(x) node_find_one(x$node, x$doc, xpath = xpath, nsMap = ns))
-  xml_nodeset(lapply(nodes, function(y) xml_node(y, x[[1]]$doc)))
+  xml_nodeset(lapply(x, function(x)
```

Comment:
The duplication check is now moved to `xml_nodeset()` in https://github.com/jimhester/xml2/commit/ab8eaffda06a49a3dcc3bb6c93e2373af865d667